website/storybook.js

```diff
@@ -0,0 +1,52 @@
+const fs = require('node:fs');
```

Comment:
should it be using typescript rather than js as it seems docusaurus can use typescript for configuration files

Reply:
Yeah, maybe, I will try,  but this is a plugin so need to do some testing

Reply:
✅ done